Resources/config/yui.yml

```diff
@@ -0,0 +1,22 @@
+system:
+    default:
+        yui:
+            modules:
+                ez-contenteditformview:
+                    requires:
+                        - 'ez-tweet-editview'
```

Comment:
you are not supposed to write this (ie redefine a PlatformUI module). In an extension bundle, you should use `dependencyOf` while defining your own modules, so here, that would look like:
```yml
ez-tweet-editview:
    requires: ['ez-fieldeditview', 'event-valuechange', 'tweeteditview-ez-template']
    dependencyOf: ['ez-contenteditformview]
    path: bundles/ezsystemstweetfieldtype/js/views/fields/ez-tweet-editview.js
ez-tweet-view:
    requires: ['ez-fieldview', 'tweetview-ez-template', 'ez-asynchronousview']
    dependencyOf: ['ez-rawcontentview']
    path: bundles/ezsystemstweetfieldtype/js/views/fields/ez-tweet-view.js
```